application/Models/AmazonProduct.cs

```diff
@@ -14,7 +14,10 @@ public class AmazonProduct
 
         //[Required]
         public Int32 Quantity { get; set; }
```

Comment:
Should this be marked as ignored by EF with `[NotMapped]`?  It looks like the field for the DB is `IsAvailable`, which derives from this.